services/httpd/response_logger.go

```diff
@@ -112,10 +114,9 @@ func buildLogLine(l *responseLogger, r *http.Request, start time.Time) string {
 		detect(referer, "-"),
 		detect(userAgent, "-"),
 		r.Header.Get("Request-Id"),
-		fmt.Sprintf("%s", time.Since(start)),
-	}
-
-	return strings.Join(fields, " ")
+		// response time, report in microseconds because this is consistent
+		// with apache's %D parameter in mod_log_config
+		strconv.FormatInt(time.Since(start).Nanoseconds()/1000, 10))
```

Comment:
I still think that `time.Since(start)/time.Microsecond` would be clearer than the static `1000`